Cargo.toml

```diff
@@ -99,6 +99,7 @@ default-members = [
 
 [profile.release]
 debug = true
```

Comment:
Unrelated to this change but do we test perf with debug=true?

Reply:
This means including debug symbols in the binary, for example you get more useful backtrace when something panics, so it's essential for debugging. The size of the binary will be bigger, but it shouldn't really affect the code being executed.